docs/source/upgrade/upgrade-guide-from-monitoring-4.x-to-monitoring-4.y.rst

```diff
@@ -24,6 +24,15 @@ Install 4.y.b (The new version)
 
 Replace “y” with the new minor release number, for example, 4.0.1.zip
 
+.. note::
+    For versions 4.7.* and above, use the format `4.7.*.zip`, for example:
+
+    .. code-block:: bash
+
+        wget -L https://github.com/scylladb/scylla-monitoring/archive/4.7.1.zip
```

Comment:
you can change the wget command to use the -O tag, so it will work regardless of the version number